templates/base/footer.html

```diff
@@ -0,0 +1,8 @@
+<footer class="container-fluid">
+            <footer class="container-fluid">
```

Comment:
you have two footers in html, also don't forget to add a new line in the end and fix the indentation

Reply:
done!!!

Reply:
It's not done, your indentation is not fixed in this file and the other file doesn't have a new line in the end.

Reply:
![screenshot from 2018-07-05 22-01-06](https://user-images.githubusercontent.com/28941044/42322264-20a25b32-809f-11e8-8212-91928c22d6aa.png)
how is it now?

Reply:
looks good, it's not pushed yet though. You need to fix index.html as well.